rotkehlchen/exchanges/cryptocom.py

```diff
@@ -280,6 +298,50 @@ def first_connection(self) -> None:
 
         self.first_connection_made = True
 
+    def deserialize_balance(
+            self,
+            new_balance_data: dict[str, Any],
+            balance_key: str,
+            existing_balances: defaultdict['AssetWithOracles', Balance],
+    ) -> defaultdict['AssetWithOracles', Balance]:
+        """Deserialize a balance dict using the amount from the specified balance_key.
+        Returns the updated existing_balances.
+        """
+        if (instrument_name := new_balance_data.get('instrument_name')) is None:
+            log.error(f'Found {self.name} balance with no instrument_name. Should not happen.')
+            return existing_balances
+
+        try:
+            asset = asset_from_cryptocom(instrument_name)
+            if (amount := deserialize_fval(new_balance_data[balance_key])) > ZERO:
+                usd_price = Inquirer.find_usd_price(asset=asset)
+                existing_balances[asset] += Balance(
+                    amount=amount,
+                    usd_value=amount * usd_price,
+                )
```

Comment:
```suggestion
            if (amount := deserialize_fval(new_balance_data[balance_key])) > ZERO:
                existing_balances[asset_from_cryptocom(instrument_name)] += Balance(
                    amount=amount,
                    usd_value=amount * Inquirer.find_usd_price(asset=asset),
                )
```

Reply:
this updates the already passed dict since it is passed as a mutable argument. Maybe the return is not needed?